templates/connected-base.html.twig

```diff
@@ -6,13 +6,10 @@
             <img class="header__logo" src="{{ asset('build/images/logo-demoapp.svg') }}"
                  alt="{{ 'page.welcome.connect.logo_alt' | trans }}" />
             <nav class="main-navigation">
-                <a href="{{ path('products') }}" class="main-navigation__link main-navigation__link--active" >
-                    {{ 'common.header.links.catalog' | trans }}
-                </a>
-                <a href="https://marketplace.akeneo.com/extension/akeneo-demo-app" target="_blank" class="main-navigation__link" >
+                <a href="https://marketplace.akeneo.com/extension/akeneo-demo-app" target="_blank" class="main-navigation__link main-navigation__link--external" >
                     {{ 'common.header.links.help' | trans }}
                 </a>
-                <a href="{{ app.session.get('pim_url') }}" target="_blank" class="main-navigation__link main-navigation__link--external">
+                <a href="{{ app.session.get('pim_url') }}/connect/apps/v1/catalogs/{{ catalog.id }}" target="_blank" class="main-navigation__link main-navigation__link--external">
```

Comment:
The link is incorrect when you look a the ticket:
```suggestion
                <a href="{{ app.session.get('pim_url') }}/#/connect/connected-apps/{{ connected_app.id }}" target="_blank" class="main-navigation__link main-navigation__link--external">
```

I don't know if we store the connected app id somewhere, maybe it should be hard-coded as it is a fixed id coming from the app store (you may have to dig a little bit on that)